src/js/helpers/apiRequest.js

```diff
@@ -20,12 +20,20 @@ export const apiRequest = (axiosParams = {}) => {
         return defaultHeaders;
     };
 
-    const params = (options = {}) => {
-        const parameters = { ...defaultParams };
-        Object.assign(parameters, options);
-        parameters.headers = headers(options.headers);
-        return parameters;
-    };
+    const params = (options = {}) => Object
+        .keys(options)
+        .reduce((acc, key) => {
+            if (key === 'data' && Object.keys(options.data).includes('award_id')) {
+                return {
+                    ...acc,
+                    data: {
+                        ...options.data,
+                        award_id: decodeURIComponent(options.data.award_id)
+                    }
+                };
+            }
+            return { ...acc, [key]: options[key] };
+        }, { ...defaultParams, headers: headers(options.headers) });
```

Comment:
Personally, I would add a `post` method, considering we would want to check all our posts if there is an `award_id`.

Something like:
```
const post = (parameters) => {
  const newParams = params(parameters);
  /**
  * Removes the encoding of the award_id.
  */
  if (newParams.data && newParams?.data.award_id) {
    // maybe check here to make sure it is encoded?
    newParams.data.award_id = decodeURIComponent(options.data.award_id);
  }
  Axios.request(newParams);
};
```
This way we will have isolated new defaults for our posts, and in the future we can do so with our gets, puts, etc...

Although, we would need to update the post routes to be posts. But since post is not default any post we have should be searchable by `method: 'post'`.  

I originally was going to set it up this way but there was no real need for it.